### ISSUE TITLE
`duplicate-names-in-diff-context` complains about void attributes

### ISSUE THREAD

**Issue by h1alexbel** [objectionary/lints#559]
Consider this EO snippet:

```eo
# Foo.
[] > foo
  [b] > bar
  [b] > xyz
```

However, I see this:

```text
Object b has same name as object on line 3 (duplicate-names-in-diff-context (Single))
Object b has same name as object on line 4 (duplicate-names-in-diff-context (Single))
```

I believe, we should not complain about void attributes, they can share same name.

**Comment by h1alexbel** [comment]
@yegor256 WDYT?

**Comment by yegor256** [comment]
@h1alexbel it's a valid complaint. Having two `b` objects in the same file may lead to a confusion. The `duplicate-names-in-diff-context` was created exactly to catch such situations.

**Comment by 0crat** [comment]
@h1alexbel Hey there! 👋 Thanks for submitting the ticket, even though it wasn't accepted this time. As per our policy, there's a -16 point deduction, but don't let that discourage you! Your contributions are valuable, and your current balance is still a solid [+255](https://www.eolang.org/zerocracy/objectionary-vitals.html). Keep reporting [bugs](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) and suggesting improvements – it's how we grow! Just be a bit more careful next time. Remember, [bugs are welcome](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html), so keep those tickets coming!
